plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php

```diff
@@ -71,6 +71,12 @@ protected function doDispatch(KalturaBatchJob $job, KalturaIntegrationJobData &$
 	
 	protected function doClose(KalturaBatchJob $job, KalturaIntegrationJobData &$data, KalturaVoicebaseJobProviderData $providerData)
 	{
+		$entryId = $providerData->entryId;
+		KalturaLog::debug("checking entry id - $entryId");
+		$this->clientHelper = VoicebasePlugin::getClientHelper($providerData->apiKey, $providerData->apiPassword);

```

Comment:
@shpiNi  I don't think there is really a point in this static function. The VoiceBaseClientHelper is not a singleton. You only instantiate it in 2 places in the code. Just call the class constructor.(Aesthetic comment, not urgent for Tue deployment)